### DISCUSSION TITLE
IPv4 주소 체계, CIDR과 서브네팅 이해하기

### DISCUSSION THREAD

**Discussion by yoondgu** [woowacourse-study/2023-tal-study#26] [General]
지난 번 토미 강의 들은 이후 서브네팅에 대해 이해하면서 정리한 내용 공유합니다.
마침 이번 강의 내용과 겹쳐서.. ㅎㅎ 블로그 홍보는 아닙니다 ^_^

https://yoondgu.tistory.com/25
https://yoondgu.tistory.com/26

**Comment by zillionme** [comment]
와 정리 진짜 잘되어있네여 짱짱.. 저는 뭘 discussion에 남겨야 할지

**Comment by yeonkkk** [comment]
갓도영.
갓도이.
그저갓.
